athena-federation-integ-test/src/main/java/com/amazonaws/athena/connector/integ/IntegrationTestBase.java

```diff
@@ -240,7 +240,7 @@ protected void cleanUp()
     public List<String> listTables(String databaseName)
             throws RuntimeException
     {
-        String query = String.format("show tables in %s.%s;", lambdaFunctionName, databaseName);
+        String query = String.format("show tables in `%s`.`%s`;", lambdaFunctionName, databaseName);
```

Comment:
does backtick actually work? I though either not enclosed, or enclosed in double quotes.

Reply:
Only for DDL commands: `lambda:function`.`database`.`table`